concepts/recursion/about.md

```diff
@@ -0,0 +1,102 @@
+# About
+
+Recursive functions are functions that call themselves.
+
+A recursive function needs to have at least one _base case_ and at least one _recursive case_.
+
+A _base case_ returns a value without calling the function again.
+A _recursive case_ calls the function again, modifying the input so that it will at some point match the base case.
+
+In Elm, recursive functions are typically defined using pattern matching with `case` expressions.
+
+```elm
+count : List a -> Int
+count list =
+    case list of
+        [] ->
+            0  -- base case
+
+        _ :: tail ->
+            1 + count tail  -- recursive case
+```
+
+A recursive function can have many base cases and/or many recursive cases.
+For example [the Fibonacci sequence][fibonacci] is a recursive sequence with two base cases:
```

Comment:
I think this is a famously slow way to solve the fibonacci sequence. Probably worth mentioning that or choosing a different example?

Reply:
I couldn't find a different example that was as simple as this one and not terribly slow, so I added a note.